api/index.js

```diff
@@ -136,26 +143,39 @@ router.use('/search', function(req, res, next) {
 });
 
 router.use('/twitter', function(req, res, next) {
-
-    const query = req.query
-    let client = new Twitter(TWITTER_API)
-    if (!('screen_name' in query) || query.screen_name === '') {
-        res.send('empty screen_name')
-    } else {
-        client.get('statuses/user_timeline', query, function(err, data) {
-            if (err) {
-                res.send(err)
-                // res.status(500).end('Internal Error 500')
-                console.error(`error during fetch data from twitter : ${req.url}`)
-                console.error(err) 
-                process.exit(1)               
-            } else {
-                res.json(data)
-            }
-        })
-    }
+  const query = req.query
+  let client = new Twitter(TWITTER_API)
+  if (!('screen_name' in query) || query.screen_name === '') {
+      res.send('empty screen_name')
+  } else {
+      client.get('statuses/user_timeline', query, function(err, data) {
+          if (err) {
+              res.send(err)
+              // res.status(500).end('Internal Error 500')
+              console.error(`error during fetch data from twitter : ${req.url}`)
+              console.error(err) 
+              process.exit(1)               
+          } else {
+              res.json(data)
+          }
+      })
+  }
 });
 
+router.use('/tracking', function(req, res, next) {
+  const query = req.query
+  const log = loggingClient.log(GCP_STACKDRIVER_LOG_NAME)
+  const metadata = { resource: { type: 'global' } }
+  const entry = log.entry(metadata, query)
+  log.write(entry).then(() => {
```

Comment:
it's async actually. And if wanna handle the error, just add some codes like following:
```
mmLog({
      eventType: 'click', // optional, default = 'click'
      target: event.target, // optional, default = undefined
      description: '' // optional, default = ''
   }).then((log) => {
      return store.dispatch('LOG_CLIENT', { params: {
            clientInfo: log
      }})
   }).catch((err) => {
      // handle error
   })
```